pkg/apis/carto/v1alpha1/cluster_config_template_test.go

```diff
@@ -17,12 +17,11 @@ package v1alpha1_test
 import (
 	"encoding/json"
 
+	"github.com/vmware-tanzu/cartographer/pkg/apis/carto/v1alpha1"
```

Comment:
the is out of order

Reply:
yeah, interesting how it is preserving the notion of local imports but it's enforcing having it in this specific location